Makefile.kokkos-kernels

```diff
@@ -27,6 +27,7 @@ KOKKOSKERNELS_INTERNAL_ENABLE_MAGMA := $(strip $(shell echo $(KOKKOSKERNELS_ENAB
 KOKKOSKERNELS_INTERNAL_ENABLE_CHOLMOD := $(strip $(shell echo $(KOKKOSKERNELS_ENABLE_TPLS) | grep "cholmod" | wc -l))
 KOKKOSKERNELS_INTERNAL_ENABLE_SUPERLU := $(strip $(shell echo $(KOKKOSKERNELS_ENABLE_TPLS) | grep "superlu" | wc -l))
 KOKKOSKERNELS_INTERNAL_ENABLE_LAPACKE := $(strip $(shell echo $(KOKKOSKERNELS_ENABLE_TPLS) | grep "lapacke" | wc -l))
```

Comment:
> Should we remove lapacke and cblas ?

SPTRSV code is using LAPACKE and CBLAS defines. I would remove LAPACKE and CBLAS when we update the SPTRSV code to use `KokkosBlas::{trmm,trtri}` -- @srajama1: shall I update SPTRSV as part of this PR? I think @iyamazaki is working on SPTRSV right now so we may get ourselves into some conflicts if I update SPTRSV in this PR.

> @e10harvey And why would we need LAPACK? Shouldn't KOKKOSKERNELS_INTERNAL_ENABLE_BLAS be enough since you put both `trmm` and `trtri` in `KokkkosBlas` namespace?

@vqd8a: Where `trtri` is supported confused me. LAPACK supports `trtri` on the host while MAGMA supports `trtri` on the device. BLAS does not support `trtri`.  So for `trtri` TPL specializations we may use LAPACK or MAGMA. This is unfortunate as LAPACK and MAGMA are additional dependencies.

Reply:
I just afraid we have too many macros. Usually, MKL, OpenBLAS and MAGMA provide blas functions and lapack functions in a single library. Correct me if I am wrong.